l1-contracts/contracts/state-transition/chain-deps/facets/Admin.sol

```diff
@@ -126,10 +131,6 @@ contract AdminFacet is ZKChainBase, IAdmin {
 
     /// @inheritdoc IAdmin
     function setPubdataPricingMode(PubdataPricingMode _pricingMode) external onlyAdmin onlyL1 {
-        // Validium mode can be set only before the first batch is processed
```

Comment:
Removing this will allow even permanent rollups to set the feeparams.pubdataPricingMode to Validium. This does not change DA, but it does change L1->L2 txs fees in Mailbox _deriveL2GasPrice , the pubdataPriceBaseToken is now not added to fullPubdataPriceBaseToken. 

This could make L1->L2 txs fail. 

Not a big issue, just trying to think it through

Reply:
makes sense, agree